airflow/requirements.txt

```diff
@@ -1,4 +1,5 @@
-/tmp/airflow_provider_weaviate-0.0.1-py3-none-any.whl
+#/tmp/airflow_provider_weaviate-0.0.1-py3-none-any.whl
```

Comment:
If no code is using it, should we just remove it? if not, should we add it back?

Reply:
removed